sudoers/files/sudoers

```diff
@@ -22,6 +23,7 @@
     {%- set runas_list_defaults = defaults.get('runas_list', {}) %}
     {%- set users = sudoers.get('users', {}) %}
     {%- set groups = sudoers.get('groups', {}) %}
+    {%- set network_groups = sudoers.get('network_groups', {}) %}
```

Comment:
Same for here adn several other sets, do we need `+` signs for the logic portions?